jira_agile_metrics/config_test.py

```diff
@@ -89,17 +88,16 @@ def test_config_to_options_maximal():
 
     Backlog column: Backlog
     Committed column: Committed
-    Final column: Test
```

Comment:
And again - changes a test that would have passed previously.

Reply:
`Final column` is not used in the code, therefore it would be okay to leave that there. However, I think it's confusing to have code or config that isn't used.